src/amo/components/UserProfile/styles.scss

```diff
@@ -0,0 +1,48 @@
+@import "~amo/css/inc/vars";
+@import "~core/css/inc/mixins";
+@import "~ui/css/vars";
+
+$avatar-size: 64px;
+
+.UserProfile {
+  margin: $padding-page;
+}
+
+.UserProfile-wrapper {
+  display: flex;
+}
+
+.UserProfile-user-info {
+  @include respond-to(large) {
+    min-width: 300px;
+    width: 35%;
+  }
+}
+
+.UserProfile-avatar {
+  display: block;
+  height: $avatar-size;
+  margin: 0 0 12px;
+  width: $avatar-size;
+}
+
+.UserProfile-name {
+  @include font-regular();
+
+  font-size: $font-size-l;
+  margin: 0;
+
+  @include respond-to(medium) {
+    font-size: $font-size-xl;
+  }
+}
+
+.UserProfile-rating-average .Rating {
+  @include margin(0, 0, 0, -3px);
```

Comment:
Seeing a `-3px` negative margin in a review is hard because I can't tell if the final margin follows our standard spacing. Maybe `Rating` needs a new class that can be toggled on via configuration? I see that search results need to adjust it too. No need to address it here but maybe file as a code quality issue.

Reply:
Good call. I think it might be that in most places it indents itself too far over; in reality I think it needs a class for the original place it was used as I think the CSS was too coupled to the parent component originally. I'll file the follow-up issues, cheers 👍

Reply:
Filed #4573.